javascriptv3/example_code/dynamodb/src/ddbdoc_get_item.ts

```diff
@@ -0,0 +1,65 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+
+ABOUT THIS NODE.JS EXAMPLE: This example works with AWS SDK for JavaScript version 3 (v3),
+which is pending release.  The preview version of the SDK is available
+at https://github.com/aws/aws-sdk-js-v3. This example is in the 'AWS SDK for JavaScript v3 Developer Guide' at
+https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/dynamodb-example-document-client.html.
+
+Purpose:
+ddbdoc_get_item.ts demonstrates how to use DynamoDB utilities to retrieve a set of attributes for an item in an Amazon DynamoDB table..
+
+Inputs (replace in code):
+- TABLE_NAME
+- REGION
+- primaryKey
+- sortKey (only required if table has sort key)
+- VALUE_1: Value for the primary key (The format for the datatype must match the schema. For example, if the primaryKey is a number, VALUE_1 has no inverted commas.)
+- VALUE_2: Value for the primary key (The format for the datatype must match the schema.)
```

Comment:
'VALUE_1' and 'VALUE_2' here but only 'VALUE' later in this file.

Reply:
implemented